diagnostic.md

```diff
@@ -38,8 +40,9 @@ $base-margin: 1.5em;
 ```
 
 ```scss
-// your response here
-```
+// li {
+  base-margin:($base-margin, 50%);
```

Comment:
Remember, we want to actually do some math here , but it's as simple as:

```css
li {
  margin: $base-margin / 2;
}
```

You're right about getting stuck on the syntax, but you're really close and just need some more practice 👍